scripts/run-ci.sh

```diff
@@ -32,6 +32,12 @@ if [ ! -z "$CPPLINT" ]; then
     exit 0
 fi
 
+if [ ! -z "$CPPLINT" ]; then
+    echo "Running gofmt ..."
```

Comment:
This must be included in the `if [ ! -z "$CPPLINT" ]; then` block above (before the `exit 0`), otherwise it will not be reached.